backend/src/main/java/team31/codesuqad/sidedish/service/EventService.java

```diff
@@ -1,87 +1,40 @@
+import org.springframework.transaction.annotation.Transactional;
+import team31.codesuqad.sidedish.controller.dto.EventTabListDto;
+import team31.codesuqad.sidedish.controller.dto.CategoryResponse;
 import team31.codesuqad.sidedish.domain.*;
-import team31.codesuqad.sidedish.repository.EventTabsRepository;
+import team31.codesuqad.sidedish.repository.EventTabRepository;
+import team31.codesuqad.sidedish.type.EventType;
 
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
-import java.util.Map;
-import java.util.function.Function;
-import java.util.stream.Collectors;
+import java.util.NoSuchElementException;
 
 @Service
+@Transactional(readOnly = true)
 public class EventService {
 
-    private EventTabsRepository eventTabsRepository;
-    private DishesService dishesService;
-    private DiscountPoliciesService discountPoliciesService;
+    private final EventTabRepository eventTabRepository;
+    private final DishService dishService;
 
-    public EventService(EventTabsRepository eventTabsRepository, DishesService dishesService, DiscountPoliciesService discountPoliciesService) {
-        this.eventTabsRepository = eventTabsRepository;
-        this.dishesService = dishesService;
-        this.discountPoliciesService = discountPoliciesService;
+    public EventService(EventTabRepository eventTabRepository, DishService dishService) {
+        this.eventTabRepository = eventTabRepository;
+        this.dishService = dishService;
     }
 
-    public EventTabResponse getEventTab(Event event) {
-        EventTabs eventTab = eventTabsRepository.findByName(event.name());
-        List<Dishes> allDishes = dishesService.findAll();
-
-        List<Dishes> dishes = findSameEventTab(allDishes, eventTab.getId());
-        mappingDiscountPolicies(dishes);
-        Collections.shuffle(dishes);
-        dishes = checkNumber(dishes);
-
-        List<DishDto> dishDtos = makeDishDto(dishes);
-        return new EventTabResponse(eventTab, dishDtos);
-    }
-
-    private List<Dishes> findSameEventTab(List<Dishes> allDishes, Integer eventId) {
-        return allDishes.stream()
-                .filter(dish -> dish.isSameEvent(eventId))
-                .collect(Collectors.toList());
-    }
-
-    private void mappingDiscountPolicies(List<Dishes> dishes) {
-        Map<Integer, DiscountPolicies> discountPoliciesMap = discountPoliciesService.findAll().stream()
-                .collect(Collectors.toMap(DiscountPolicies::getId, Function.identity()));
-
-        for (Dishes dish : dishes) {
-            List<DiscountPolicies> discountPolicies = dish.getDiscounts().stream()
-                    .map(Discount::getDiscountPolicyId)
-                    .map(discountPoliciesMap::get)
-                    .collect(Collectors.toList());
-            dish.setEventBadge(discountPolicies);
-        }
+    public EventTabListDto findAll() {
+        List<Event> events = eventTabRepository.findAll();
+        return new EventTabListDto(events);
```

Comment:
👍 
아래와같이 지역변수 할당 없이 사용 가능할듯 합니다 :)
```suggestion
    public EventTabListDto findAll() {
        return new EventTabListDto(eventTabRepository.findAll(););
```